app/blog/shared/BlogWrapper.js

```diff
@@ -31,24 +32,74 @@ export default function BlogWrapper({
         </div>
       </div>
 
-      <div className={styles.sectionsWrapper}>
-        <div className={styles.sections}>
-          <Link href="/blog" className="inline-flex" aria-label="Blog Homepage">
-            <AiOutlineHome size={18} />
+      <div
+        className={
+          sectionSlug ? styles.sectionsSlugWrapper : styles.sectionsWrapper
+        }
+      >
+        <div className={sectionSlug ? styles.sectionsSlug : styles.sections}>
+          <Link
+            id="blog-home"
+            href="/blog"
+            className="inline-flex"
+            aria-label="Blog Homepage"
+          >
+            <BaseButtonClient
+              style={{ backgroundColor: '#000', color: '#ffffff' }}
+              className="py-3 px-4 mb-3 font-bold"
+            >
+              <AiOutlineHome size={18} />
+            </BaseButtonClient>
           </Link>
           {sections.map((section) => (
             <React.Fragment key={section.fields.slug}>
               {section.fields.slug === sectionSlug ? (
-                <div className={`${styles.section} ${styles.active}`}>
+                // <div className={`${styles.section} ${styles.active}`}>
+                //   {section.fields.title}
+                // </div>
+                <BaseButtonClient
+                  className={`${
+                    section.fields.slug == 'onboarding-live'
+                      ? 'bg-indigo-900'
+                      : section.fields.slug == 'politics'
+                      ? 'bg-violet-600'
+                      : section.fields.slug == 'the-independent-cause'
+                      ? 'bg-pink-600'
+                      : section.fields.slug == 'temp-section'
+                      ? 'bg-orange-600'
+                      : section.fields.slug == 'candidates'
+                      ? 'bg-teal-500'
+                      : 'bg-gray-800'
+                  } py-3 px-4 mb-3 ml-3 font-bold text-white rounded-full`}
+                >
                   {section.fields.title}
-                </div>
```

Comment:
I see you have been defining this in multiple places.
Let's create a component for this blog button